package.json

```diff
@@ -5,9 +5,11 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "start:firefox": "web-ext run --source-dir ./src/"
+    "start:firefox": "web-ext run --source-dir ./src/",
```

Comment:
Is there a `-t firefox`, just like for Chromium? If so, maybe we should be explicit here.